lib/scss_lint/linter/single_line_per_property.rb

```diff
@@ -4,6 +4,8 @@ class Linter::SingleLinePerProperty < Linter
     include LinterRegistry
 
     def visit_rule(node)
+      visit_children(node)
```

Comment:
This can just be `yield`, to match what we do elsewhere. It's perhaps not as descriptive but at least consistent. I usually write:

```ruby
yield # Continue linting children
```